unittests/linuxliketests.py

```diff
@@ -1759,16 +1759,13 @@ def test_as_link_whole(self):
             self.assertNotIn('-lfoo', content)
 
     def test_prelinking(self):
-        # Prelinking currently only works on recently new GNU toolchains.
-        # Skip everything else. When support for other toolchains is added,
-        # remove limitations as necessary.
-        if 'clang' in os.environ.get('CC', 'dummy') and not is_osx():
-            raise SkipTest('Prelinking not supported with Clang.')
         testdir = os.path.join(self.unit_test_dir, '86 prelinking')
         env = get_fake_env(testdir, self.builddir, self.prefix)
         cc = detect_c_compiler(env, MachineChoice.HOST)
         if cc.id == "gcc" and not version_compare(cc.version, '>=9'):
             raise SkipTest('Prelinking not supported with gcc 8 or older.')
+        if cc.id == "clang" and not version_compare(cc.version, '>=14'):
```

Comment:
```suggestion
        if cc.id == 'clang' and not version_compare(cc.version, '>=14'):
```

Be consistent.

Reply:
It's consistent with the above line, though ;)